lib/src/disposable_manager/disposable_manager_vm.dart

```diff
@@ -14,7 +14,7 @@
 
 import 'dart:async';
 
-import 'package:w_common/disposable.dart';
+import 'package:w_common/disposable_vm.dart';
```

Comment:
In general I'm against using our own top-level imports in `src/` files, and this is more relevant now since we could get to a point in the future where the `disposable_vm.dart` entry point may actually include some `dart:io` functionality which would break this. This could be replaced with `src/` imports for the individual pieces that are needed